### DISCUSSION TITLE
Required argument in hybrid_command

### DISCUSSION THREAD

**Discussion by p-krystian** [Rapptz/discord.py#8370] [Q&A]
### Is a way to mark an argument in `hybrid_command` as required?

In case
```python
@commands.hybrid_command(name='examle')
async def _example(ctx, arg:str=''):
    pass
```
or
```python
@commands.hybrid_command(name='examle')
async def _example(ctx, arg:typing.Optional[str]):
    pass
```
the argument is not required in a slash command and in case
```python
@commands.hybrid_command(name='examle')
async def _example(ctx, arg:str):
    pass
```
is required, but I getting error when I am using this command by prefix like `?example`.
**I just need to mark the argument as required only when using slash command**

**Comment by Rapptz** [comment]
There is no way to do this with hybrid commands since the idea behind them is to share the same interface for both.

**Reply by p-krystian** [reply]
Ok, thanks
